frontend/src/About.js

```diff
@@ -0,0 +1,20 @@
+import React, {useState, useEffect} from 'react';
+import logo from './logo.png';
+import './menu.css'
+
+function About() {
+    const [about, setAbout] = useState("About");
+
+    useEffect(() => {
+        fetch("/about").then(response => response.json()).then(data => {setAbout(data.about)})
+    }, [])
+
+    return (
+        <div>
+            <img src={logo} className="logo" />
```

Comment:
Remove the logo from here otherwise it will be repeated in the website. (it appears in Menu and in About)